CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.9.1 (2020-02-07)
+
+- Fixed several deep imports into other SKY UX packages to address build warnings when using Angular Ivy. [#52](https://github.com/blackbaud/skyux-i18n/pull/52)
```

Comment:
What is a deep import? If it's common knowledge, then I think it's fine as is, but if not, we should tweak the description.

Reply:
It's fairly common: https://stackoverflow.com/a/46446124/6178885

Reply:
Cool, just wanted to check in before approving if it was something that could be misunderstood. I could've googled that...haha, sorry.